server/controllers/profile.js

```diff
@@ -0,0 +1,55 @@
+const router = require('express').Router();
+let Profile = require("../models/Profile");
+const { route } = require('../routes/auth');
+
+router.route('/').get((req, res) => {
+    Profile.find()
+      .then(profiles => res.json(profiles))
+      .catch(err => res.status(400).json('Error: ' + err));
+});
+
+router.route('/create').post((req,res) => {
+    const firstName = req.body.firstName;
+    const lastName = req.body.lastName;
+    const description = req.body.description;
+    const profilePic = req.body.profilePic;
+    const availability = req.body.availability;
+
+    const newProfile = new Profile ({
+        firstName,
+        lastName,
+        description,
+        profilePic,
+        availability,
+    });
+
+    newProfile.save()
+        .then(() => res.json("Profile Created"))
+        .catch(err => res.status(400).json('Error: ' + err))
```

Comment:
Ditto throughout, we should preference `async/await` and ensure we are  using the `express-async-handler` let's look at the other controllers as an example.